src/controllers/update/update.director.check.answers.controller.ts

```diff
@@ -33,6 +34,11 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     const session: Session = req.session as Session;
 
     const officerFiling = await getOfficerFiling(session, transactionId, submissionId);
+
+    // redirect to update page if no changes have been made
+    if (!officerFiling.nameHasBeenUpdated && !officerFiling.nationalityHasBeenUpdated && !officerFiling.occupationHasBeenUpdated && !officerFiling.correspondenceAddressHasBeenUpdated && !officerFiling.residentialAddressHasBeenUpdated) {
```

Comment:
If we are thinking reusability it would make more sense as middleware to me. I don't see it as validation as such, just a navigational switch. Happy to discuss

Reply:
It's a bit long as a test and will need to be added to if there are any further fields to fill in (yes, I know that's a paper form change too) so I think worthwhile encapsulating. It a sort of validation check in that in order to file the user must have made at least one change.

Reply:
Middleware is fine too.